internal/aws/k8s/k8sclient/obj_store.go

```diff
@@ -0,0 +1,156 @@
+// Copyright  OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package k8sclient
+
+import (
+	"fmt"
+	"sync"
+
+	"go.uber.org/zap"
+	"k8s.io/apimachinery/pkg/api/meta"
```

Comment:
I'm trying to find this package but can't find it, do you know why? I found this one.

https://pkg.go.dev/k8s.io/apimachinery/pkg/apis/meta/v1